eth-types/src/geth_types.rs

```diff
@@ -145,6 +171,25 @@ impl Transaction {
     }
 }
 
+impl From<crate::Transaction> for Transaction {
```

Comment:
nitpick: define `From<crate::Transaction>` seems redundant. We have `From<&crate::Transaction> ` already, so that we can use `Transaction::from(&tx)` everywhere.

Reply:
Fixed in b25452b

Reply:
Fixed in b25452b